src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,261 @@
+  registration.showNotification(title, {
+    tag: tag,
+    body: "This notification was scheduled 30 seconds ago",
+    showTrigger: new TimestampTrigger(timestamp + 30 * 1000)
+  });
+};
+```
+
+{% Aside %}
+  On desktop, Notifications Triggers will fire only if Chrome is running. On Android,
+  Notifications Triggers will fire completely independent from Chrome.
+{% endAside %}
+
+### Canceling a previously scheduled notification
+
+To cancel previously scheduled notifications, first request a list of all notifications that match a
+certain tag through `ServiceWorkerRegistration.getNotifications()`. Note that you need to pass the
+`includeTriggered` flag for scheduled notifications to be included in the list.
+
+```js
+const tag = 'meeting';
+const cancelScheduledNotification = async (tag) => {
+  const registration = await navigator.serviceWorker.getRegistration();
+  const notifications = await registration.getNotifications({
+    tag: tag,
+    includeTriggered: true,
+  });
+
+  if (Array.isArray(notifications)) {
+    notifications.forEach((notification) => notification.close());
+  }
+};
+```
+
+### Debugging
+
+You can use Chrome DevTools' Notifications debug service on the Application tab. To start debugging,
+press the small "Start recording events" button in the top-left corner (the circle), or hit
+'Command/Control' + 'E'. This allows you to both examine scheduled notifications, as well as see
+displayed and closed notifications. The screenshots below show this in action.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="devtools-scheduled.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Chrome DevTools showing the "scheduled" state of a notification.
+  </figcaption>
+</figure>
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="devtools-displayed.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 2 — Chrome DevTools showing the "displayed" state of a previously scheduled notification.
+  </figcaption>
+</figure>
+
+### Demo
+
+You can see Notification Triggers in action in the [demo][demo], which allows you to schedule
+notifications, list scheduled notifications, and cancel them. The source code is available on
+[Glitch][demo-source].
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="demo.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 3 — Notification Triggers <a href="https://notification-triggers.glitch.me/">demo</a>.
+  </figcaption>
+</figure>
+
+## Security and permissions
+
+We've designed and implemented Notification Triggers using the core principles defined in
+[Controlling Access to Powerful Web Platform Features][powerful-apis], including user control,
+transparency, and ergonomics. As this API depends on service workers, it is only available in secure
+contexts. Using the API requires the same permission as regular push notifications.
```

Comment:
I don't think this line is as clear as it could be. Try:

'Because this API requires service workers, it also requires a secure context.'

Reply:
Done.